omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/CreateUpdatePatientResource1_9Test.java

```diff
@@ -73,4 +75,18 @@ public void shouldUpdatePatient() throws Exception {
 		Assert.assertEquals("Race = Muslim", attributes.get(0).get("display"));
 	}
 	
+	@Test
+	public void shouldCreatePatient_fromGET() throws Exception {
+		executeDataSet("personAttributeTypeWithConcept.xml");
+		SimpleObject patientSimpleObject = new SimpleObject();
+		InputStream object = getClass().getClassLoader().getResourceAsStream("create_patient_from_get.json");
```

Comment:
One of the reasons this test fails is because of the test patient data in this json file, this is a 1.9 test yet the test json patient data includes fields that were added in later versions hence the failure.